package.json

```diff
@@ -9,7 +9,14 @@
     "_metalsmith": "bin/_metalsmith"
   },
   "scripts": {
-    "test": "make test"
+    "pretest": "npm install",
+    "test": "mocha $MOCHA_OPTS",
+    "pretest-debug": "npm install",
+    "test-debug": "mocha debug",
+    "precoverage": "npm install",
+    "coverage": "istanbul cover ./node_modules/.bin/_mocha",
```

Comment:
@Ajedi32 yep, just realized that myself. The problem here is `--harmony_generators` is only supported by `mocha` while istanbul needs to be called with `_mocha`.

But shouldn't it actually be sufficient to run the coverage with only one node version?